src/std/deprecation.ss

```diff
@@ -0,0 +1,18 @@
+;;; -*- Gerbil -*-
+;;; © vyzo
+;;; package deprecation warnings
+
+(export #t)
+
+(defsyntax (package-deprecation-warning! stx)
+  (syntax-case stx ()
+    ((_)
+     (let* ((package (expander-context-id (current-expander-context)))
+            (msg (call-with-output-string ""
+                   (lambda (out)
+                     (display "*** WARNING package :" out)
+                     (display package out)
+                     (display " is DEPRECATED and will be removed in the future" out)
+                     (newline out)))))
```

Comment:
Maybe add optional note for what it is being replaced by and/or which documentation to consult about that?

Reply:
Also, a per-function or per-binding deprecation system would be nice.

Reply:
per-function we can do when we need it, pretty easy.

Reply:
uhm maybe, it's a nit.